### DISCUSSION TITLE
Data and Access

### DISCUSSION THREAD

**Discussion by jalpa015** [arun-esh/travelx#59] [Access and Permissions]
Request for link to Data Source and description

**Comment by arun-esh** [comment]
At this point, we are using airbnb's open dataset along with mongoDB. I'll create a user for you to access the database.

**Comment by arun-esh** [comment]
@jalpa015 User has been created and information has been sent in an email. We can also think about other dataset if you prefer.

**Comment by arun-esh** [comment]
@jalpa015 I have added you as collaborator in Heroku account. Create an account, if you don't have, with your Uwindsor email. Please let me know if you don't have access to my account. We will setup a meeting.

**Reply by jalpa015** [reply]
@arun-esh 
Thank you. All set with Data and Heroku access.
